ipaserver/install/ipa_restore.py

```diff
@@ -423,6 +423,12 @@ def run(self):
                 logger.info('Restarting SSSD')
                 sssd = services.service('sssd', api)
                 sssd.restart()
+                logger.info('Restarting oddjobd')
+                oddjobd = services.service('oddjobd', api)
+                if not oddjobd.is_enabled():
+                    logger.info("Enabling oddjobd")
+                    oddjobd.enable()
+                oddjobd.restart()
```

Comment:
Hi @slaykovsky 
No need to restart oddjobd if it is already running. You can replace this call with oddjobd.start()